CMakeLists.txt

```
@@ -57,7 +57,9 @@ project(${BOARD_LIB_NAME}
         LANGUAGES CXX C
         )
 
-add_library(${PROJECT_NAME} STATIC)
+add_library(${PROJECT_NAME} STATIC
+        include/INA138.hpp
+        src/INA138.cpp)
 
 # Add sources
 target_sources(${PROJECT_NAME} PRIVATE
```

This is where you want to add the file, and you'll only want to put the src file
```suggestion
target_sources(${PROJECT_NAME} PRIVATE
        src/INA138.cpp
```